install.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ "$(uname)" == "Darwin" ]; then
+  config_dirs_template="$HOME/Library/Preferences/XXX*"
+else
+  config_dirs_template="$HOME/.XXX*/config"
```

Comment:
What if someone has a hidden folder for dirty things?

I think you should use something like `JETBRAINSAPPFOLDER` as a placeholder instead :)